webservices/common/models/auditsearch.py

```diff
@@ -0,0 +1,33 @@
+
+from .base import db, BaseModel
+from webservices import docs
+
+class AuditSearchMixin(object):
+    __table_args__ = {"schema": "auditsearch"}
+
+class AuditCase(db.Model, AuditSearchMixin):
```

Comment:
Just a note that we already went over [Python's MRO and order of mixins](https://stackoverflow.com/questions/10018757/how-does-the-order-of-mixins-affect-the-derived-class); will be fixed in a future commit to this PR!